cmake/multi_image.cmake

```diff
@@ -56,6 +56,11 @@ function(add_child_image)
   set(oneValueArgs NAME SOURCE_DIR DOMAIN)
   cmake_parse_arguments(ACI "" "${oneValueArgs}" "" ${ARGN})
 
+  if (NOT CONFIG_PARTITION_MANAGER_ENABLED)
+    message(FATAL_ERROR
+      "Please make sure that CONFIG_PARTITION_MANAGER_ENABLED is set")
```

Comment:
Add description on how to set the kconfig correctly.

Reply:
Build strategy template.

Reply:
Give the user information about which child image did not set the config, so that they know where it should be fixed.

Check if we can perform a separate check for each image in addition.

Reply:
See jira 6992